.github/workflows/cd-teardown.yml

```
@@ -0,0 +1,33 @@
+name: CD
+
+on: delete
+
+env:
+  CI: true
+jobs:
+  unpublish-npm:
+    runs-on: ubuntu-18.04
```

Might as well use the latest version if we're setting this up anew :)

```suggestion
    runs-on: ubuntu-20.04
```